### PR TITLE
Education form

### PR DESCRIPTION
**Pull request by Ngai-E**
Ok i have redesigned this @muarachmann  hope this is better.

### CODE REVIEW

**Comment by Ngai-E** [comment]
hope no one is one this? I just noticed the sqlconf file again. Re-sending the PR.

**Comment by KoniKodes** [comment]
@muarachmann @VirginiaBalseiro Can we look at this please?
Thank you.

**Comment by KoniKodes** [comment]
Hi. What is the status on this? Is it ready to merge?

**Comment by Ngai-E** [comment]
I think it is.

**Comment by KoniKodes** [comment]
I thought so too...

@muarachmann @VirginiaBalseiro @robbyoconnor 
Can we please have eyes and approval on this? Then can we please merge it? 
It plays an important role in Maggies internship and j-pro's volunteer projects.
Thank you all for responding....

**Comment by muarachmann** [comment]
Okay will have a look over the weekend and get back to you

**Comment by KoniKodes** [comment]
Thank you. Have a great weekend.

**Comment by muarachmann** [comment]
Tested functioning LGTM :)

File: interface/forms/education/new.php [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+			let topic = newTopicHtml();
+			$('.addTopic').prepend(topic);
+		}
+		function inactivateTopic(TopicNumber) {
+			console.log(TopicNumber);
+			$(`.status_${TopicNumber}`).attr("value", "0");
+			$(`.labelTopic_${TopicNumber}`).html("Topic Completed");
+		}
+		function inactivateLearner(issueNumber, TopicNumber) {
+			var learner = $(`#learner_${issueNumber}${TopicNumber}`);
```
Comment (muarachmann) [inline interface/forms/education/new.php]: using `let` to be consistent

**Comment by KoniKodes** [comment]
Thank you @muarachmann

**Pull request merged**